backup

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+export PATH=$PATH:/usr/local/bin
```

Comment:
There's no necessity to add `/usr/local/bin` to the `PATH`. No program is called in this script by its name, *except* `borg`. So adding a `CMD_BORG` variable would be a proper solution and even a great feature!

Reply:
Please get used to call scripts by its command name and not by its PATH. Sometimes, there are good reasons to use full paths, but most of the time it's an antipattern.

Reply:
That's fair. If that's the case, though, the Makefile needs to be re-written as well because that's where the executables reside.

Reply:
I should mention that NOT putting the path in the script will cause RHEL6+ and Debian Wheezy+ to maybe not find the Borg executable on a system that's been locked down. Scripts executed by cron/anacron don't end up having a pre-defined PATH variable and the backup script will fail.